rudder-web/src/main/scala/bootstrap/liftweb/AppConfigAuth.scala

```diff
@@ -134,6 +186,27 @@ class AppConfigAuth extends Loggable {
     }
   }
 
+  @Bean def fileAuthenticationProvider : AuthenticationProvider = {
```

Comment:
So you can see what you need to either defined a code-based provider, or a file based one. And you get a tast of the horrors of spring.